client/src/components/MusicController/MusicController.tsx

```diff
@@ -0,0 +1,123 @@
+import {
+  BsFillPlayCircleFill,
+  BsFillPauseCircleFill,
+  BsShuffle,
+} from 'react-icons/bs';
+import { CgPlayTrackNext, CgPlayTrackPrev } from 'react-icons/cg';
+import { FiRepeat } from 'react-icons/fi';
+import { useAppDispatch } from 'src/store/hooks';
+import {
+  fetchNextOrPrevTrack,
+  fetchPauseOrPlay,
+  fetchRepeat,
+  fetchShuffle,
+  setMusicIsPlaying,
+  setRepeatState,
+  setShuffleState,
+  setTrackTrigger,
+} from 'src/store/musicPlayerSlice';
+import usePlaylist from 'src/utils/usePlaylist';
+import {
+  ControllerContainer,
+  ShuffleWrapper,
+  PrevWrapper,
+  PlayPauseWrapper,
+  NextWrapper,
+  RepeatWrapper,
+} from '../../styles/MusicControllerStyle';
+
+function MusicController() {
+  const { musicIsPlaying, shuffleState, repeatState } = usePlaylist();
+  const dispatch = useAppDispatch();
+
+  // onClick to change track w/ prev/next buttons
+  const changeTrack = async (prevOrNext: string) => {
+    await dispatch(fetchNextOrPrevTrack(prevOrNext)) // trigger next/prev song
+      .then((status) => {
+        if (status.meta.requestStatus === 'fulfilled') {
+          dispatch(setTrackTrigger(true));
+        }
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  // onClick to pause or play track
+  const mpState = async (curState: string) => {
+    let boolState;
+    await dispatch(fetchPauseOrPlay(curState))
+      .then((status) => {
+        console.log(status);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+    if (curState === 'play') {
+      boolState = true;
+    } else {
+      boolState = false;
+    }
+    dispatch(setMusicIsPlaying(boolState));
```

Comment:
Might be nice to streamline (and get rid of the `boolState` var altogether)?
```suggestion
    dispatch(setMusicIsPlaying(curState === 'play'));
```